madmom/features/beats.py

```diff
@@ -1147,7 +1147,7 @@ def process_forward(self, activations, reset=True, **kwargs):
             sys.stderr.write('\r%s' % ''.join(display))
             sys.stderr.flush()
         # forward path often reports multiple beats close together, thus report
-        # only beats more than the minumum interval apart
+        # only beats more than the minimum interval apart
```

Comment:
Please bundle these two fixes in a single commit with a meaningful commit message, e.g. 'fix typos'.